osbenchmark/aggregator.py

```diff
@@ -1,39 +1,41 @@
+import os
 from typing import Any, Dict, List, Union
 import uuid
 
 from osbenchmark.metrics import FileTestExecutionStore
 from osbenchmark import metrics, workload, config
+from osbenchmark.utils import io as rio
 
 class Aggregator:
-    def __init__(self, cfg, test_executions_dict):
+    def __init__(self, cfg, test_executions_dict, args):
         self.config = cfg
+        self.args = args
         self.test_executions = test_executions_dict
         self.accumulated_results: Dict[str, Dict[str, List[Any]]] = {}
         self.accumulated_iterations: Dict[str, int] = {}
+        self.statistics = ["throughput", "latency", "service_time", "client_processing_time", "processing_time", "error_rate", "duration"]
```

Comment:
Nit: we use `metrics` instead of `statistics`, which would standardize on official documentation terminiology: https://opensearch.org/docs/latest/benchmark/reference/metrics/metric-keys/

Reply:
Makes sense, updated this name